doc/examples/motion_planning_python_api/motion_planning_python_api_tutorial.rst

```diff
@@ -0,0 +1,298 @@
+
+        plan_request_params:
+                planning_attempts: 1
+                planning_pipeline: ompl
+                max_velocity_scaling_factor: 1.0
+                max_acceleration_scaling_factor: 1.0
+
+        ompl_rrtc:
+                plan_request_params:
+                        planning_attempts: 1
+                        planning_pipeline: ompl
+                        planner_id: "RRTConnectkConfigDefault"
+                        max_velocity_scaling_factor: 1.0
+                        max_acceleration_scaling_factor: 1.0
+                        planning_time: 0.5
+
+        ompl_rrt_star:
+                plan_request_params:
+                        planning_attempts: 1
+                        planning_pipeline: ompl_rrt_star # Different OMPL pipeline name!
+                        planner_id: "RRTstarkConfigDefault"
+                        max_velocity_scaling_factor: 1.0
+                        max_acceleration_scaling_factor: 1.0
+                        planning_time: 1.5
+
+        pilz_lin:
+                plan_request_params:
+                        planning_attempts: 1
+                        planning_pipeline: pilz_industrial_motion_planner
+                        planner_id: "PTP"
+                        max_velocity_scaling_factor: 1.0
+                        max_acceleration_scaling_factor: 1.0
+                        planning_time: 0.8
+
+        chomp:
+                plan_request_params:
+                        planning_attempts: 1
+                        planning_pipeline: chomp
+                        max_velocity_scaling_factor: 1.0
+                        max_acceleration_scaling_factor: 1.0
+                        planning_time: 1.5
+
+
+The first block of the configuration file sets the planning scene monitor options such as the topics that it subsribes to (Note: if you aren't familiar with the planning scene monitor, you should consider reviewing ... ): ::
+
+        planning_scene_monitor_options:
+                name: "planning_scene_monitor"
+                robot_description: "robot_description"
+                joint_state_topic: "/joint_states"
+                attached_collision_object_topic: "/moveit_cpp/planning_scene_monitor"
+                publish_planning_scene_topic: "/moveit_cpp/publish_planning_scene"
+                monitored_planning_scene_topic: "/moveit_cpp/monitored_planning_scene"
+                wait_for_initial_state_timeout: 10.0
+
+The second block of the configuration file sets the planning pipelines that we wish to use. MoveIt supports multiple motion planning libraries including OMPL, PILZ industrial motion planner, Stochastic Trajectory Optimization for Motion Planning (STOMP), Search-Based Planning Library (SBPL) and Covariant Hamiltonian Optimization for Motion Planning (CHOMP) to name a few. When configuring our ``moveit_py`` node, we need to specify the configuration for planning pipelines we wish to use: ::
```

Comment:
Is there a link to a doc section listing all the planners we can throw here?

Maybe https://moveit.picknik.ai/humble/doc/examples/examples.html#configuration